

global_frame: odom
robot_base_frame: base_link
update_frequency: 0.5
publish_frequency: 1.0 # for visualization

rolling_window: true
resolution: 0.05
width: 3
height: 3
origin_x: 0.0
origin_y: 0.0

plugins:
  - {name: inflation_local, type: "costmap_2d::InflationLayer"}
  - {name: obstacles_layer,           type: "costmap_2d::ObstacleLayer"}
